documentation/docs/scenarios/xsa-deploy/Readme.md

```diff
@@ -0,0 +1,96 @@
+# Build and Deploy SAPUI5 or SAP Fiori Applications on SAP Cloud Platform with Jenkins
+
+Build an application based on SAPUI5 or SAP Fiori with Jenkins and deploy the build result into an SAP Cloud Platform account in the Neo environment.
+
+## Prerequisites
+
+* You have installed the Java Runtime Environment 8.
+* You have installed Jenkins 2.60.3 or higher.
+* You have set up Project “Piper”. See [README](https://github.com/SAP/jenkins-library/blob/master/README.md).
+* You have installed the Multi-Target Application (MTA) Archive Builder 1.0.6 or newer. See [SAP Development Tools](https://tools.hana.ondemand.com/#cloud). TODO: maybe we should refer to the docker image
+* You have installed Node.js including node and npm. See [Node.js](https://nodejs.org/en/download/). TODO: docker ?
+
```

Comment:
In case they are available on SMP: should we also explain how the artifacts needs to be provided in the build environment of the customer. Here I see basically two possibilities:

1.) local m2 repo
2.) nexus on customer side

Ad 1.) that was state of the art before docker. With docker we have to mount the local repo in each docker container performing a java build.

Ad 2.) Nexus can be setup based on a docker image.  Needs to be included in `settings.xml`.

Regardless which option we recommand we should also explain how the artifacts can be provided there (e.g. `install:install-file`, `deploy:deploy-file`. For nexus there is also a bulk upload available (for nexus OS a corresponding plugin needs to be installed first). Bulk upload into a local repo can be performed by expanding a suitable `tar.gz`

Reply:
We will have Java dependencies on maven by end of december, so that is the default case.

For the non-default (SMP) my suggestion would be 

> In case you require dependencies that are not available on maven central, you will need to download them available at build-time, preferably by deploying them to your local artifact repository (e.g. nexus).